app/templates/events/view/speakers.hbs

```diff
@@ -0,0 +1,98 @@
+<div class="ui grid stackable">
+  <div class="row">
+    <div class="twelve wide column">
+      <div class="ui basic buttons">
+        <button class="ui button">{{t 'All'}}</button>
+        <button class="ui button">{{t 'Pending'}}</button>
+        <button class="ui button">{{t 'Accepted'}}</button>
+        <button class="ui button">{{t 'Rejected'}}</button>
+      </div>
+      <button class="ui teal button">{{t 'Add Speaker'}}</button>
+      <button class="ui teal button">{{t 'Export as CSV'}}</button>
+      {{#ui-dropdown class='labeled icon button'}}
+        <i class="dropdown icon"></i>
+        <div class="default text">{{t 'No. of entries'}}</div>
+        <div class="menu">
+          <div class="item">{{t '10'}}</div>
+          <div class="item">{{t '25'}}</div>
+          <div class="item">{{t '50'}}</div>
+          <div class="item">{{t '100'}}</div>
+        </div>
+      {{/ui-dropdown}}
+    </div>
+    <div class="four wide column right aligned">
+      {{#ui-search }}
+        <div class="ui icon input">
+          <input class="prompt" type="text" placeholder="Search Speakers...">
+          <i class="search icon"></i>
+        </div>
+      {{/ui-search}}
+    </div>
+  </div>
+  <div class="row">
+    <div class="sixteen wide column">
+      <table class="ui stackable very basic table">
+        <thead>
+          <tr>
+            <th></th>
+            <th>{{t 'Name'}}</th>
+            <th>{{t 'Email'}}</th>
+            <th>{{t 'Phone'}}</th>
+            <th>{{t 'Submitted Sessions'}}</th>
+            <th>{{t 'State'}}</th>
+            <th></th>
+          </tr>
+        </thead>
+        <tbody>
+          {{#each model as |speaker|}}
+            <tr>
+              <td class="right aligned">
+                <div>
+                  <img class="ui mini avatar round image" alt="speakers_image" src={{speaker.image}}>
```

Comment:
Or use the safe image component. Would automatically load a palceholder when image doesn't exisit.